tests/test_model_builder.py

```diff
@@ -257,12 +257,14 @@ def test_fit_no_t(toy_X):
     assert "posterior" in model_builder.idata.groups()
 
 
-@pytest.mark.xfail
-def test_fit_dup_Y(toy_X):
-    # create redundant target column in X
+def test_fit_dup_Y(toy_X, toy_y):
+    # Create redundant target column in X
     toy_X = pd.concat((toy_X, toy_y), axis=1)
     model_builder = ModelBuilderTest()
-    model_builder.fit(X=toy_X, chains=1, draws=100, tune=100)
+
+    # Expecting an exception due to redundant target columns
+    with pytest.raises(ValueError, match="Redundant target column found in input data."):
```

Comment:
I think we need to change the `match` statement, as the test are failing:

```python
    def test_fit_dup_Y(toy_X, toy_y):
        # Create redundant target column in X
        toy_X = pd.concat((toy_X, toy_y), axis=1)
        model_builder = ModelBuilderTest()
    
        # Expecting an exception due to redundant target columns
>       with pytest.raises(ValueError, match="Redundant target column found in input data."):
E       AssertionError: Regex pattern did not match.
E        Regex: 'Redundant target column found in input data.'
E        Input: "X includes a column named 'output', which conflicts with the target variable."
```

Reply:
ok let me check